onnxruntime/test/providers/cpu/controlflow/if_test.cc

```diff
@@ -73,7 +74,18 @@ class IfOpTester : public OpTester {
       inputs = {split_input};
       outputs = {&split_out_0, &split_out_1};
 
-      graph.AddNode("split", "Split", "Split into 2", inputs, outputs);
+      auto& split_node = graph.AddNode("split", "Split", "Split into 2", inputs, outputs);
+      if (opset_version_ == 11) {
```

Comment:
>opset_version_ == 11 [](start = 10, length = 20)

> 10?

Reply:
we don't need it for 10 right ? In opset 10, we need to preserve the "even" split, only opset 11 and above supports unequal shapes in both the branches...

Reply:
Oh I see what you mean now. You wanted `== 11` changed to `> 10`. Will do.

Reply:
Fixed.